### PR TITLE
Add the new platform ACP_6_3

### PR DESCRIPTION
**Pull request by saisurya-ch**
Add the new platform support for ACP_6_3.

### CODE REVIEW

**Review by lgirdwood — COMMENTED** [review]
@thesofproject/amd pls review.

**Review by marc-hb — DISMISSED** [review]

File: scripts/xtensa-build-all.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ set -e
 # They must have a toolchain available in the latest Docker image.
 DEFAULT_PLATFORMS=(
     imx8 imx8x imx8m imx8ulp
-    rn rmb vangogh
+    rn rmb vangogh acp_6_3
```
[the remark this answers is not in the document]
Comment (saisurya-ch) [reply scripts/xtensa-build-all.sh]: Done

File: scripts/xtensa-build-all.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@ DEFAULT_PLATFORMS=(
 
 # Work in progress can be added to this "staging area" without breaking
 # the -a option for everyone.
-SUPPORTED_PLATFORMS=( "${DEFAULT_PLATFORMS[@]}" )
+SUPPORTED_PLATFORMS=( "${DEFAULT_PLATFORMS[@]}" acp_6_3 )
```
Comment (marc-hb) [inline scripts/xtensa-build-all.sh:18]: I meant `SUPPORTED_PLATFORMS+=( mt8188 acp_6_3 )` of course :-(

Comment (marc-hb) [reply scripts/xtensa-build-all.sh:18]: Or even better:

```shell
# Waiting for container work in progress
SUPPORTED_PLATFORMS+=( mt8188 )

# No plan to support in container (or replace with the appropriate comment)
SUPPORTED_PLATFORMS+=( acp_6_3 )
```

**Comment by keqiaozhang** [comment]
SOFCI TEST

File: scripts/xtensa-build-all.sh (hunk 2 above)
Comment (saisurya-ch) [reply scripts/xtensa-build-all.sh]: I have made the changes accordingly.

thanks @marc-hb

**Review by bhiregoudar — APPROVED** [review]

**Comment by kv2019i** [comment]
Ignoring "sof-ci/jenkins/pr-tools-build" (transient error still present, checking build manually), proceeding with merge.

**Pull request merged**